packages/main/src/Table.js

```diff
@@ -49,6 +49,28 @@ const metadata = {
 		},
 	},
 	properties: /** @lends sap.ui.webcomponents.main.Table.prototype */ {
+
+		/**
+		 * Defines the text that will be displayed when there is no data and <code>show-no-date</code> is present.
+		 *
+		 * @type {String}
+		 * @public
```

Comment:
add ```@defaultvalue: ""```

Reply:
And in the same sense (as the other comment), we describe the types as "boolean" and "string" (instead of Boolean and String), so if you can do it ```@type {string}```. Thanks.